src/pages/EditMonitor.vue

```diff
@@ -48,6 +48,11 @@
                         <input type="number" class="form-control" id="interval" v-model="monitor.interval" required min="20" step="1">
                     </div>
 
+                    <div class="mb-3">
+                        <label for="maxRetries" class="form-label">Retry Pings</label>
```

Comment:
Looing for better name for this, "Retry Pings" looks strange to me.

Reply:
Just `Retries`?
And IMO will be nice to also add some "tip" (outside `<label>`) like: "Maximum retries before sending a notification"

Reply:
Ye I'm not very good at naming things.
Retries sounds a lot better.

As for the hint, how about "Maximum retries before service is marked as down"?

Reply:
LGTM